views/styles/chart_styles.js

```diff
@@ -149,31 +149,32 @@ export const chartStyles = (theme) => ({
     justifyContent: 'space-between',
   },
   filterFormContainer: {
-    paddingBottom: '25px',
+    paddingBottom: '75px',
   },
   filters: {
-    width: '100%',
+    width: '35%',
```

Comment:
IMO we shouldn't use percent widths. Is there a reason we can't provide a specific width value in pixels?